starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,50 @@
 // Names and Input
+var hacker1 = "Name";
+console.log("the driver name is " + hacker1);
 
 
+var hacker2= prompt("what is your user name ?");
+console.log("the navigator's name is " + hacker2);
+
 //Conditionals
+if (hacker1.length > hacker2.length){
+    console.log("The Driver has the longest name, it has " + hacker1.length +" characters");
+  }
+  else if (hacker2.length > hacker1.length){
+    console.log("Yo, navigator got the longest name, it has " + hacker2.length + " characters");
+  }
+  else{
+    console.log("wow, you both got equally long names, "+ hacker1.length + " characters!!");
+  }
+
+var hacker1Upp = hacker1.toUpperCase();
+console.log(hacker1Upp);
+console.log(hacker1Upp.split('').join(' '));
+
+console.log(hacker1.split("").reverse().join(""));
+
+var lexicoSort = [hacker1, hacker2];
+lexicoSort.sort();
+console.log(lexicoSort);
+if (lexicoSort[0] = hacker1 ){
```

Comment:
this if statement is not working, if I enter Marie, I still get "the driver goes first" :
that is because you are using  "=" (affectation operator) in your condition instead of "===" (equality operator). This is a very common mistake !